### PR TITLE
fix: missing dep on ipython_genutils

### LINKED ISSUE #1725: No module named ipython_genutils
Recently I had readthedocs break with `No module named ipython_genutils` coming from nbconvert. Looking at the code in nbconvert, I clearly see `ipython_genutils` used in a few places, but no dependence on `ipython_genutils` declared in setup.py.

RtD log: https://readthedocs.org/projects/hist/builds/16329065/

<details>
<summary>Inline log error:</summary>

```pytb
Running Sphinx v4.4.0
loading translations [en]... done

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/sphinx/registry.py", line 433, in load_extension
    mod = import_module(extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/nbsphinx.py", line 42, in <module>
    import nbconvert
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/nbconvert/__init__.py", line 4, in <module>
    from .exporters import *
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/nbconvert/exporters/__init__.py", line 3, in <module>
    from .html import HTMLExporter
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/nbconvert/exporters/html.py", line 25, in <module>
    from nbconvert.filters.highlight import Highlight2HTML
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/nbconvert/filters/__init__.py", line 11, in <module>
    from ipython_genutils.text import indent
ModuleNotFoundError: No module named 'ipython_genutils'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/sphinx/cmd/build.py", line 280, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/sphinx/application.py", line 230, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/sphinx/application.py", line 387, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/hist/envs/387/lib/python3.8/site-packages/sphinx/registry.py", line 436, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname,
sphinx.errors.ExtensionError: Could not import extension nbsphinx (exception: No module named 'ipython_genutils')

Extension error:
Could not import extension nbsphinx (exception: No module named 'ipython_genutils')
```

</details>

Manually adding this to my requirements fixes the build. https://github.com/scikit-hep/hist/pull/387/commits/611c452c88866131acb1080b3c0f821aa29f61d7

**Nbconvert version: 6.4.2**

### PR DESCRIPTION
**Pull request by henryiii**
Fixes #1725.

See https://github.com/jupyter/nbconvert/search?q=ipython_genutils for usages.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -224,7 +224,8 @@ def get_data_files():
     'pandocfilters>=1.4.1',
     'testpath',
     'defusedxml',
-    'nbclient>=0.5.0,<0.6.0'
+    'nbclient>=0.5.0,<0.6.0',
+    'ipython_genutils',
```
Comment (henryiii) [inline setup.py:228]:
```suggestion
    'ipython-genutils',
```

Could write it this way instead.

**Review by bollwyvl — APPROVED** [review]
this looks like the correct play for now

**Comment by davidbrochart** [comment]
I [dropped ipyton_genutils in nbconvert](https://github.com/jupyter/nbclient/pull/209), since it was mostly used to handle python2/python3 compatibility. Maybe we can do the same here, instead of depending on it?

**Comment by davidbrochart** [comment]
mmm it seems it's not only about py2/3 compat here, so maybe it's fine to still depend on it.
I don't like the about section of ipython_genutils though: "Vestigial IPython utilities: DO NOT USE" :smile:

**Comment by blink1073** [comment]
Closing in favor of #1727.  Thanks @henryiii!

**Pull request closed without merging**